packages/smooth_app/lib/pages/user_management/sign_up_page.dart

```diff
@@ -371,7 +371,7 @@ class _SignUpPageState extends State<SignUpPage> with TraceableClientMixin {
       return;
     }
     if (status.error != null) {
```

Comment:
Not a fan of this method, what if u enter everything correctly and there's an actual server error that won't let you sign up
Stil the user would be shown, 'Enter valid email', which is surely not right,

I guess we should be dealing with it in off-dart,
@monsieurtanuki  should we create an issue for this there?

Reply:
The topic of the error is inside `statusErrors`, as shown in the `contains` statement just afterwards - this is where we should compute the message from. The default would be something like `'Server error'`.

Reply:
Yes @monsieurtanuki but status.error message is a long html reponse which probably is a issue from server side code , And instead of "Enter Valid Email" We can display 'Server error' @AshAman999 What do you propose

Reply:
@Akashsri3bi What I meant was something like 
```dart
final String message;
if (status.statusErrors!.contains(SignUpStatusError.EMAIL_ALREADY_USED)) {
   message = 'Email already used';
} ...
```

And probably not using `status.error` at all.

Reply:
Ohk I think I got your point  , Thanks @monsieurtanuki !